src/hooks/useVirtualization.hook.js

```diff
@@ -71,10 +71,124 @@ export const getRowsSizes = (
   })
 }
 
+/**
+ * Hook for scrolling a table to a selected item.
+ * @param {Object} options - Options object.
+ * @param {React.RefObject} options.tableRef - Ref object for the table element.
+ * @param {Object[]} options.content - Array containing content for each row.
+ * @param {Object{}} options.selectedItem - Object representing the currently selected item.
+ * @param {Object} options.expandedRowsData - Object containing data for expanded rows.
+ * @param {string|number} options.rowHeight - Height of a regular row.
+ * @param {string|number} options.rowHeightExtended - Height of an extended row.
+ * @param {string|number} options.headerRowHeight - Height of an table header.
+ * @param {boolean} options.useHook - Bool indicate if trigger the hooks logic.
+ * @returns {void}
+ */
```

Comment:
don't use `options` in the JSDoc. Just to be consistent with `useVirtualization` hook